docs/pages/docs/guides/auth-and-access-control.mdx

```diff
@@ -0,0 +1,532 @@
+
+The equivalent GraphQL query would look like this:
+
+```graphql
+query {
+  person(where: { id: $session.itemId }) {
+    name
+    isAdmin
+  }
+}
+```
+
+## Adding Access Control
+
+Now that we have information about the current user, lets put it to good use and add some security to our API with **Access Control**.
+
+The way to think about access control is –
+
+- With **Session Data**, you know who the user is, and what they should have access to (whether that's permissions granted by a role, or items related to them)
+- For each **List** you can restrict which operations they can perform, filter the items they have access to, and control which fields they can read and update.
+
+!> In other words: **Step 1** load data about the current user; **Step 2** use that data to limit what they can do.
+
+You define access control for each List (and field) individually; the rules you specify will always be run, even when the list being queried or mutated through a relationship.
+
+### Posts Example
+
+Let's say we have a list of Blog Posts, which have a published status. We want Admin users to see all blog posts and be able to crete and update them, but anonymous (public) users can only see published posts and can't write to the API. Our Post list might look like this:
+
+```ts
+const Post = list({
+  fields: {
+    title: text(),
+    isPublished: checkbox(),
+    publishDate: timestamp(),
+    author: relationship({ ref: 'Person' }),
+    // more content fields would go here
+  },
+});
+```
+
+And the Session Data we set up above would look like this:
+
+```ts
+type Session = {
+  data: {
+    id: string;
+    isAdmin: boolean;
+  }
+}
+```
+
+We can now set up operational access control to restrict the **create**, **update** and **delete** operations to authenticated users with the `isAdmin` checkbox set:
+
+```ts
+const isAdmin = ({ session }: { session: Session }) => session?.data.isAdmin;
+
+const Post = list({
+  access: {
+    operation: {
+      create: isAdmin,
+      update: isAdmin,
+      delete: isAdmin,
+    },
+  },
+  fields: {
+    // see above
+  },
+});
+```
+
+We can also use **filter** access control to make sure that unauthenticated users can only see published posts:
+
+```ts
+const filterPosts = ({ session }: { session: Session }) => {
+  // if the user is an Admin, they can access all the records
+  if (session?.data.isAdmin) return true;
+  // otherwise, filter for published posts
+  return { isPublished: { equals: true } };
+}
+
+const Post = list({
+  access: {
+    operation: {
+      // see above
+    },
+    filter: {
+      query: filterPosts,
+    }
+  },
+  fields: {
+    // see above
+  },
+});
+```
+
+See that object we return when the session data doesn't say there's a user with `isAdmin`? It's the same format as `where` arguments you can provide to the `posts` query through the GraphQL API:
```

Comment:
Not a change, but tonality check here. "See that object we return when the session data doesn't say there's a user with `isAdmin`?" is a tone of voice we haven't used anywhere else in the docs iirc, is consistency of tone something we care about across docs? If yes, might be worthwhile rewriting this phrase.

Reply:
Fair call. I think I'm trying to make the guide generally more conversational, but tbh it's all feeling a bit wordy at the moment. I've re-written this more consistently for now 👍